picoGPT/demo.py

```
@@ -0,0 +1,120 @@
+DEFAULT_NUM =3 
+
+
+def add_numbers(num1, num2):
+    num1 = DEFAULT_NUM + num1
+    return num1 + num2
+
+
+class Operaror:
+    @classmethod
+    def add_numbers(cls, num1, num2):
+        num1 = set_random(num1)
```

Ruff (0.8.2)

Undefined name `set_random`

(F821)